crates/orderbook/src/arguments.rs

```diff
@@ -120,6 +120,10 @@ pub struct Arguments {
     /// to get them errors and our liveness check fails.
     #[clap(long, default_value = "24")]
     pub solvable_orders_max_update_age_blocks: u64,
+
+    /// Enable limit orders. Once the full limit order flow is implemented, this can be removed.
+    #[clap(long, env, default_value = "false")]
```

Comment:
```suggestion
    #[clap(long, env]
```
iirc bool arguments don't need a default because the absence of the env variable or command line argument is already false. Probably doesn't hurt either.